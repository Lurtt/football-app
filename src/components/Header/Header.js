import React from 'react'
import styled from 'styled-components'
import PropTypes from 'prop-types'

import Headline from '../Headline'

const HeaderStyle = styled.header`
  position: sticky;
  top: 0;
  background-color: #34495e;
  padding: 1rem;
  box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.2);
`

export default function Header({ team, formation }) {
  return (
    <HeaderStyle>
      <Headline
        level={2}
        uppercase
        textAlign="center"
        margin="0"
        color="#ecf0f1"
        letterSpacing="0.05rem"
      >
        {team}
      </Headline>
      <Headline level={3} textAlign="center" margin="0" color="#bdc3c7">
        Formation {formation}
      </Headline>
    </HeaderStyle>
  )
}

Header.propTypes = {
  team: PropTypes.string.isRequired,
  formation: PropTypes.string.isRequired,
}
