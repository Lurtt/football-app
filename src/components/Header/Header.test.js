import React from 'react'
import renderer from 'react-test-renderer'

import Header from './'

const data = { team: 'Dark Side', formation: '333' }

describe('<Header />', () => {
  test('Should render', () => {
    const result = renderer.create(<Header {...data} />).toJSON()
    expect(result).toMatchSnapshot()
  })
})
