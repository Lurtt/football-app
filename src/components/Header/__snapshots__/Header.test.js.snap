// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render 1`] = `
<header
  className="sc-bdVaJa dRkfdz"
>
  <h2
    className="sc-bwzfXH ddMwrD"
    letterSpacing="0.05rem"
  >
    Dark Side
  </h2>
  <h3
    className="sc-htpNat beRMen"
  >
    Formation 
    333
  </h3>
</header>
`;
